src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -152,6 +159,19 @@ public synchronized void deleteCard(Card card) throws CardNotFoundException {
         indicateAddressBookChanged();
     }
 
+    @Override
+    public void answerSelectedCard(int confidenceLevel) throws NoCardSelectedException {
+        if (selectedCard == null) {
+            throw new NoCardSelectedException();
+        } else if (confidenceLevel == 0) {
+            // repeat in ui
+        } else if (confidenceLevel == 1) {
+            selectedCard.getSchedule().feedback(false);
+        } else if (confidenceLevel == 2) {
+            selectedCard.getSchedule().feedback(true);
+        }
```

Comment:
Are we planning to remove the card from the list of cards to be reviewed every time the person answers it? In this implementation it feels like the filtered card list will not change. This also means it's not possible for line 167 to be implemented because we cant throw the card to the back of the list

Reply:
I was thinking that the UI side would have something to handle viewing of cards.

But I guess it could be implemented in the models as well. Suppose that we want to implement it in the model using this exact filtered list. Could we not just pop it out like a queue and add it to the back?